src/bp/ui-studio/src/web/views/OneFlow/index.tsx

```diff
@@ -171,15 +178,20 @@ const FlowBuilder = (props: Props) => {
         permissions={actions}
         flowPreview={flowPreview}
         onCreateFlow={createFlow}
+        selectedTopic={currentTopic}
+        selectedWorkflow={currentWorkflow}
       />
       <div className={style.diagram}>
         <Diagram
           readOnly={readOnly}
           flowPreview={flowPreview}
           showSearch={showSearch}
+          topicQnA={topicQnA}
           hideSearch={() => setShowSearch(false)}
           handleFilterChanged={handleFilterChanged}
           highlightFilter={highlightFilter}
+          selectedTopic={currentTopic}
+          selectedWorkflow={currentWorkflow}
```

Comment:
Can you put those two properties in the own component's connect instead ?

Reply:
Since I need to use the URL to know when QnA is selected, I need to pass them down as props